docs/upgrade/3.1.rst

```diff
@@ -11,57 +23,140 @@ What's new in 3.1
 Switch from MPTT to MP
 ======================
 
-Since django CMS 2.0 we relied on MPTT (Modified Preorder Tree Traversal) for efficiently handling tree structures in the DB.
-Over the years it proved that MPTT was not fast enough for big tree operations (>1000 pages) and regularly
-the tree got corrupted because of transactional errors.
+Since django CMS 2.0 we have relied on MPTT (Modified Preorder Tree Traversal) for efficiently
+handling tree structures in the database.
+
+In 3.1, `Django MPTT <https://github.com/django-mptt/django-mptt>`_ has been replaced by
+`django-treebeard <https://github.com/tabo/django-treebeard>`_, to improve performance and
+reliability.
+
+Over the years MPTT has proved not to be fast enough for big tree operations (>1000 pages); tree
+corruption, because of transactional errors, has also been a problem.
+
```

Comment:
It is my understanding that MPTT (Modified Preorder Tree Traversal) is a general logical topic and that both django-mptt and django-treebeard implement concrete solutions around this. This above reads like MPTT is a product that we have abandoned in favour of Materialized Path. I don't think this is correct. I think we should replace "MPTT" with "django-mptt" in the above paragraph, and in the one below.